packages/wordcount/package.json

```
@@ -26,6 +26,7 @@
 	"module": "build-module/index.js",
 	"react-native": "src/index",
 	"wpScript": true,
+	"types": "build-types/index.d.ts",
```

This can simply be pointed to the directory

```suggestion
	"types": "build-types",
```